packages/applications/legacy/src/modules/notification/eventHandlers/modificationRequests/handleModificationRequestStatusChanged.ts

```diff
@@ -76,6 +78,30 @@ export const handleModificationRequestStatusChanged =
             }),
           ),
         );
+
+        if (type === 'recours') {
+          const result = await deps.getUsersByRole('cre');
+
+          if (result.isOk()) {
+            await Promise.all(
+              result.value.map(({ email, fullName }) => {
+                deps.sendNotification({
```

Comment:
tu ne retournes pas la promise
```suggestion
                return deps.sendNotification({
```